styles/modules/_chat.scss

```diff
@@ -394,8 +394,9 @@
   }
 }
 
-.chatConversation .emoji, #chatContainer .emoji {
-  width: 16px;
-  height: 16px;
+.chatConversation .emoji, #chatContainer .chatConvoMessages .emoji {
+  width: 1.3em;
+  height: 1.3em;
+  vertical-align: bottom;
```

Comment:
Why is the bottom vert align necessary here but not in all the other places we use emojis?

Reply:
Because most of the other places they're center aligned using flex, here they're inline and without the bottom alignment they're really off center. 

I tried using middle instead, and it's still off due to the size of the emoji. This was the best compromise.